src/scripts/generate-binary-ops.py

```diff
@@ -0,0 +1,347 @@
+#!/usr/bin/env python
+
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import argparse
+import settings
+import collections
+path = os.path.join(settings.TEMPLATE_DIR)
+sys.path.append(path)
+import validate
+import random
+import math
+import subprocess
+
+BINARY_OPERATORS = ["+", "-", "div", "*", "%", "^", "&", "|", ">>", "<<"] # The binary operators used in this script
+DEFAULT_TEST_COUNT = 100 # The Default value of --test-count option
+OPERAND_COUNT_MIN = 2 # The --operand-count option's minimum value
+OPERAND_COUNT_MAX = 11 # The --operand-count option's maximum value
+DEFAULT_SEED = 10000 # The default value of random seed
+GENERATED_FILENAME = '''binary-ops-{NUMBER}.js''' # The file name prefix of the generated tests
+MAX = (1 << 53) - 1 # The max value of random number
+MIN = -1 * MAX  # The min value of random number
+BITMAX = 1 << 32 - 1 # The max value of random number for bitwise operations
+BITMIN = - BITMAX # The min value of random number for bitwise operations
+BITMAX_EXPOSANT = 31 # Maximum number for safely shifting an integer
+BITMIN_EXPOSANT = 0  # Because of bit operations are pecise only in 32 bit
+TEST_CASE_IN_A_FILE = 800 # The number of test cases before start new file
```

Comment:
Constants are now okay, but can we separate it to an other file? e.g `constans.py`?